### ISSUE TITLE
[Listbox] Hovered and Focused states of menu items are not styled correctly

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3258]
### 😯 Current Behavior

Current style for focused and hovered menu items are not the same as presented in Figma. Focused element is hard to distinguish from other items.

Current style for focused menu item:
<img width="1165" height="149" alt="Image" src="https://github.com/user-attachments/assets/21258e7a-d508-4898-a173-6e3a783690f9" />
<br/>
Current style for hovered menu item:
<img width="1165" height="142" alt="Image" src="https://github.com/user-attachments/assets/98e49c8b-1ba1-4dc4-b396-d1cce1bcc67c" />

### 🤔 Expected Behavior/possible solution?

Style for focused and hovered menu items should be aligned with Figma.

Focused state:
<img width="864" height="260" alt="Image" src="https://github.com/user-attachments/assets/904c70f4-ad7e-4248-8fa7-3660292847ad" />
<br/>
Hovered state: 
<img width="869" height="184" alt="Image" src="https://github.com/user-attachments/assets/0add0349-aeab-4a37-bc33-56974d0fff1b" />

### 🖥️ Steps to Reproduce

1. Go to any story that uses listbox, e.g.: https://storybook.sanomalearning.design/?path=/story/form-combobox-multiple--basic
2. Expand listbox and check if styles for focused and hovered menu items are aligned with Figma: https://www.figma.com/design/CHpKrPIdXdbV2u7X8vizKI/Components-2.0?node-id=11960-24662&t=NifkbdDKll6rfTXM-0

### Were you able to reproduce this in our storybook?

Yes

### If yes, provide the storybook URL

Any story that uses listbox, e.g.: https://storybook.sanomalearning.design/?path=/story/form-combobox-multiple--basic

### ⚖️ Impact

This blocks our release or stops us from using this component

### Framework

Other

### What browsers are you seeing the problem on?

All

### Component version

v0.1.6

### 👤 Your name

Kacper

### 🧢 Your product/team

SLDS

### 🎨 Your Theme(s)

Sanoma Learning

**Comment by DanielleRameau** [comment]
- [ ] update figma component (select and combobox)
- [ ] update code

**Comment by a11ymiko** [comment]
It's fine to use "doubled" focus rings at the same time, one for combobox and the second one as fake fokus on listbox options. Example of such an approach from ARIA APG:

<img width="425" height="407" alt="Image" src="https://github.com/user-attachments/assets/51787ab2-874d-4236-a2ff-eb8c35d968aa" />
